docs/mongodb/week-3/lesson.md

```diff
@@ -4,309 +4,142 @@ title: MongoDB - 3
 sidebar_label: Lesson
 ---
 
-Outline:
+## Outline
 
-- Lesson 2 review
-- Indexing
-  - indexing a collection
-  - indexing a collection using a collated index
-  - implementing full text search in MongoDB
-- Role based authentication
-  - cluster based roles
-  - backup and restoration roles
-- Aggregation
-- Replication
-- Transactions
-- A project in MongoDB
-- Next steps
+- This session is all about recapping the previous two weeks of MongoDB and also the Node.js API concepts.
+- The idea is to have a very hands on session that allows the learners to cement their undertsanding.
+- There may be a lot of confusion between all the different technologies and how they all fit together.
+- The lesson is designed to be run in cycles of:
+  - Thought experiment (~15 mins)
+  - Instructor led pseudo-code (~10 mins)
+  - Student led implementation (~30 mins)
+  - Intructor led implementation (~10 mins)
+- The expectation is that the whole of week 1 and week 2 was not fully completed by this point.
 
-## Review
 
-In the last lesson, we learned
+## Setup
 
-- Signing up for MongoDB Atlas
-- Making a local Express server
-- Connecting to a MongoDB server from a local Node server
-- Creating and dropping databases and collections in Atlas
-- Reading documentation
-- BSON types
-- Creating documents
-- Creating a document in Atlas
-- Creating a document with the MongoDB Node Driver
-- Updating a document
-- Deleting a document
+Learners should have already created an [Atlas account](https://www.mongodb.com/cloud/atlas), have [Compass](https://www.mongodb.com/products/compass) installed. If not you will need to allow time for them to get set up.
 
-Now that we know how to read databases, collections, and documents, we are ready to move on to more advanced tasks.
+Learners should create a database for this session, they should have a database user that can connect to this database. Students should be able to connect to this database with Compass.
 
-### Use the sample database in MongoDB Atlas
```

Comment:
## Suggest: Give ONE course of action

This section is a bit woolly and offers too much choice.  Better to dictate to teachers and TAs what to do.

I suggest we include a short exercise that takes everyone through doing exactly the same db setup, user setup, data load (if necessary).  (or point at one from a previous week, parameterised with a different db name).